src/Compilers/Core/Portable/FileSystem/PathUtilities.cs

```diff
@@ -444,5 +460,182 @@ internal static bool ContainsPathComponent(string path, string component, bool i
 
             return false;
         }
+
+        /// <summary>
+        /// Gets a path relative to a directory.
+        /// </summary>
+        public static string GetRelativePath(string directory, string fullPath)
```

Comment:
That's fine, I am mainly just interested in how the implementations differ, as a missing case (in either implementation) seems like something that should be corrected. 😄